src/Models/HydrostaticFreeSurfaceModels/HydrostaticFreeSurfaceModels.jl

```diff
@@ -86,6 +86,10 @@ Return a flattened `NamedTuple` of the prognostic fields associated with `Hydros
 @inline prognostic_fields(model::HydrostaticFreeSurfaceModel) =
     hydrostatic_prognostic_fields(model.velocities, model.free_surface, model.tracers)
 
+@inline prognostic_fields(model::HydrostaticFreeSurfaceModel{<:Any, <:Any, <:Any, <:Any, ZStarSpacingGrid}) =
+    merge(hydrostatic_prognostic_fields(model.velocities, model.free_surface, model.tracers), 
+          (; ∂t_∂s = grid.Δzᵃᵃᶠ.∂t_∂s))
```

Comment:
What does `∂t_∂s` mean?

Reply:
You probably want to define an interface to `grid.Δzᵃᵃᶠ.∂t_∂s`. This is specific to the grids we have because it requires the property `Δzᵃᵃᶠ` to exist. We discourage writing assumptive code like this.

Reply:
right, I will add that (it actually already exists, I just forgot to add it there)